yarn-project/cli/README.md

```diff
@@ -20,420 +20,4 @@ bash -i <(curl -s https://install.aztec.network)
 
 ## Usage
 
-To use `aztec-cli`, open a terminal or command prompt and run the `aztec-cli` command followed by the desired command and its options.
```

Comment:
This was super stale so I just instructed reader to print out `aztec-cli --help`